src/pencilkit.cs

```diff
@@ -246,9 +302,167 @@ interface PKToolPicker {
 		[Export ("colorUserInterfaceStyle", ArgumentSemantic.Assign)]
 		UIUserInterfaceStyle ColorUserInterfaceStyle { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Create individual instances instead.")]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Create individual instances instead.")]
 		[Static]
 		[return: NullAllowed]
 		[Export ("sharedToolPickerForWindow:")]
 		PKToolPicker GetSharedToolPicker (UIWindow window);
+
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("showsDrawingPolicyControls")]
+		bool ShowsDrawingPolicyControls { get; set; }
+
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[NullAllowed]
+		[Export ("stateAutosaveName")]
+		string StateAutosaveName { get; set; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
```

Comment:
add `[DisableDefaultCtor]` because
* there's a `[DesignatedInitializer]`
* there's no way to set the properties after creating the instance (immutable) so whatever `init` creates has very low value (if it even works)